### DISCUSSION TITLE
关于YAML解析

### DISCUSSION THREAD

**Discussion by suninx** [izhangzhihao/intellij-rainbow-brackets#2658] [Q&A] [answered]
JetBrains IDE中bundle插件YAML存在性能问题,譬如CPU高占用;
我看到本项目YAML解析使用该插件辅助解析,后续是否有自行实现YAML解析器计划?
![image](https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/42999064/05fa6c91-796e-4b34-805d-a778e0f57eff)

**Comment by izhangzhihao** [comment, marked as answer]
Hi, 本项目不会对特定语言额外再次实现解析。YAML 插件的性能问题请报告给 JetBrains：https://youtrack.jetbrains.com/issue
